tests/jax/tpu-embedding.libsonnet

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+local common = import 'common.libsonnet';
+local mixins = import 'templates/mixins.libsonnet';
+local tpus = import 'templates/tpus.libsonnet';
+
+{
+  local tpu_embedding = self.tpu_embedding,
+  tpu_embedding:: common.JaxTest {
+    local config = self,
+    frameworkPrefix: 'jax.tpu.embedding',
+    extraFlags:: [],
+    testCommand:: error 'Must define `testCommand`',
+    testScript:: |||
+      pip install --upgrade 'jax[tpu]==0.4.4' -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
+      pip install flax==0.6.7
+      gsutil cp gs://cloud-tpu-tpuvm-artifacts/tensorflow/20230214/tf_nightly-2.13.0-cp38-cp38-linux_x86_64.whl .
+      pip install tf_nightly-2.13.0-cp38-cp38-linux_x86_64.whl
+      git clone https://github.com/jax-ml/jax-tpu-embedding.git
+      %(testCommand)s
+    ||| % config.testCommand,
+  },
+  local tpu_embedding_pjit = self.tpu_embedding_pjit,
+  tpu_embedding_pjit:: tpu_embedding {
+    local config = self,
+    frameworkPrefix: 'jax.tpu.embedding',
```

Comment:
Actually you don't need to specify `frameworkPrefix`, as it will obtain from `tpu_embedding`? This comment also applies to 2nd test.